client/stylesheets/shared/_gutenberg-components.scss

```diff
@@ -5,3 +5,12 @@ Import Gutenberg component SCSS so webpack's postcss process can handle theme-in
 allows Woo themed components based on the config found in postcss.config.js
  */
 @import 'gutenberg-components/button/style.scss';
+@import 'gutenberg-components/checkbox-control/style.scss';
+@import 'gutenberg-components/dashicon/style.scss';
+@import 'gutenberg-components/form-toggle/style.scss';
+@import 'gutenberg-components/icon-button/style.scss';
+@import 'gutenberg-components/notice/style.scss';
+@import 'gutenberg-components/select-control/style.scss';
+@import 'gutenberg-components/spinner/style.scss';
+@import 'gutenberg-components/text-control/style.scss';
+@import 'gutenberg-components/tooltip/style.scss';
```

Comment:
I tried adding `@import 'gutenberg-components/form-token-field/style.scss';` because I saw form field tokens look off:

![Screen Shot 2019-07-22 at 2 40 46 PM](https://user-images.githubusercontent.com/1922453/61603458-ba755d80-ac91-11e9-8555-0c66c23192a8.png)

And it led me down a path of adding variable after variable. It reminded me that adding variables here was simply a stop gap until CSS could be exported from Gutenberg independently. Otherwise, we'll be forever configuring these. More info here, https://github.com/WordPress/gutenberg/pull/7716#issuecomment-404468856.

It seems like time to make an upstream contribution. Since we don't use these tokens (at the moment), we can leave as is.

Reply:
I'm not grokking what the upstream contribution would be to export SCSS from Gutenberg based on that PR discussion. Is the change to move the styling to the JS files?

Reply:
No, but that was a potential solution. `postcss-loader` uses the `postcss.config.js` file to apply Woo branding to styles in Gutenberg repository. For example, here is a style from [form token field](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/form-token-field/style.scss#L90):

```css
.components-form-token-field__token-text {
    background: transparent;
    color: theme(secondary);
}
```

We need to explicitly `@import 'gutenberg-components/form-token-field/style.scss'` so that our build step can process the `theme` function call and apply Woo's `secondary` color. This requires that we also import all scss variables also found in the file, which is what this PR does.

At the very least, Gutenberg will export all its variables, includes, and mixins in a package so that we don't have to go through this process on every update to `@wordpress/components` and instead consume the exported variables at build time. At the very best, our build step builds and themes css at build time without us having to do anything, as would be the case with css in js.